### PR TITLE
Fixes #15793: Add a section in plugin documentation about plugin installation

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/15793

### CODE REVIEW

File: src/reference/modules/plugins/pages/index.adoc [hunk 1]
```diff
@@ -45,4 +45,13 @@ See all available commands with:
 /opt/rudder/bin/rudder-pkg --help
 ----
 
+== Install your licenses
+
+Some plugins needs a license to work properly. All you licenses are signed with a key (license.key file)
```
Comment (Fdall) [inline src/reference/modules/plugins/pages/index.adoc]: your

File: src/reference/modules/plugins/pages/index.adoc [hunk 2]
```diff
@@ -45,4 +45,13 @@ See all available commands with:
...
+and there is one file per plugin needing a license (<plugin-name>.license). You may have downloaded them from https://download.rudder.io/plugins/licenses
```
Comment (Fdall) [inline src/reference/modules/plugins/pages/index.adoc]: per licensed plugin ?

File: src/reference/modules/plugins/pages/index.adoc [hunk 3]
```diff
@@ -45,4 +45,13 @@ See all available commands with:
...
+
+You need to put all files in `/opt/rudder/etc/plugins/licenses` folder, if that directory does not exist please create it.
+
+If the plugin is installed, no need to restart Rudder, it will read the new license and check it compatibility directly
```
Comment (Fdall) [inline src/reference/modules/plugins/pages/index.adoc]: its compatibility immediately ?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**